conf/test_full.config

```diff
@@ -18,7 +18,7 @@ params {
     input = '${projectDir}/tests/csv/3.0/test_full_data.csv'
 
     // Other params
-    tools = 'strelka,freebayes,haplotypecaller,vep'
+    tools = 'strelka,freebayes,haplotypecaller,deepvariant,manta,tiddit,ascat,snpeff,vep'
```

Comment:
ASCAT only works for paired samples at the moment., cnvkit however is also implemented for germline only mode

Reply:
Do we really want to do annotation? The containers are huge

Reply:
well it's a full test so it shouldn't matter much, but maybe just VEP for now then